packages/starlight-showcases/components/ShowcaseProfileCard.astro

```diff
@@ -0,0 +1,76 @@
+---
+import type { HTMLAttributes } from 'astro/types'
+import { Image } from 'astro:assets'
+
+export type ShowcaseProfileCardProps = Props
+
+interface Props extends Omit<HTMLAttributes<'a'>, 'title'> {
+  description?: string
+  name: string
+  picture: string
+}
+
+const { name, description, picture, ...attributes } = Astro.props
+---
+
+<div class="starlight-showcases-profile-card">
+  <span class="sl-flex stack">
+    <a {...attributes}>
+      <span class="title" set:html={name} />
+    </a>
+    {description && <span class="description" set:html={description} />}
```

Comment:
For "stack" and "description", I didn't if you preferred to keep them with their original `<LinkCard>` naming or use a `starlight-showcases-profile-*` naming.